src/main/kotlin/data/dataStore/CategoriesDataStore.kt

```diff
@@ -0,0 +1,8 @@
+package org.qudus.squad.data.dataStore
+
+interface CategoriesDataStore {
```

Comment:
You may refactor the interface name
```suggestion
interface FinanceTrackerDataStore {
```